src/shared/zustand/event/index.js

```diff
@@ -11,7 +11,22 @@ const initialState = {
   upComingEvents: [],
   onGoingEvents: [],
   historyEvents: [],
-};
+  currentEventInfo: {
+    hostId: 0,
+    participants: [
```

Comment:
Better save the data into a separate file instead of hardcoding here. it helps you to quickly switch to the actual API we are using at a later stage.